Source/Dafny/AST/Printer.cs

```diff
@@ -2870,6 +2870,14 @@ public class Printer {
             wr.Write(")");
           }
           break;
+        case DisjunctivePattern dp:
+          var psep = "";
```

Comment:
Seems like we have a difference in style preference. I always write names out in full, so here it would be `pattern` or `disjunctivePattern` and `patternSeparator`. Maybe we can agree on that we should optimise for reading? Would that change your preference?

Reply:
In general I find the shorter names more readable:

```
(-b + sqr(b*b - 4*a*c)) / (2 * a)
vs
(-linearFactor +- squareRoot(linearFactor * linearFactor - 4 * quadraticFactor * constantFactor)) / (2 * quadraticFactor)
```

In the snippet you highlighted (where there's a single variable) I don't have strong preferences; I just followed the style of the code right above.

For externally visible parameters (like function arguments or class fields) I prefer long names.